pilot/cmd/sidecar-injector/main.go

```diff
@@ -203,6 +205,8 @@ func init() {
 		"File containing the Istio mesh configuration")
 	rootCmd.PersistentFlags().StringVar(&flags.injectConfigFile, "injectConfig", "/etc/istio/inject/config",
 		"File containing the Istio sidecar injection configuration and template")
+	rootCmd.PersistentFlags().StringVar(&flags.injectValuesFile, "injectValues", "/etc/istio/inject/values",
+		"File containing the Istio sidecar injection values, in yaml format")
```

Comment:
Would it be hard to reload this file ( and the template ) - periodically is probably fine.

Reply:
Good call, I thought that everything was actually read at inject time, but looking closer we actually have a periodic reloading. Values file is read each time we inject, so may cause a minor performance issue. We can do something similar here, should be pretty simple. Thanks for pointing this out